qa/common/sequence_util.py

```diff
@@ -63,7 +63,7 @@
 
 _deferred_exceptions_lock = threading.Lock()
 _deferred_exceptions = None
-_jetson_slowdown_factor = 1.6
+_jetson_slowdown_factor = 2
```

Comment:
You should ideally be able to modify just this variable and the sequence thresholds should be adjusted by this on jetson